plugins/c8y_log_plugin/src/main.rs

```diff
@@ -87,7 +88,8 @@ async fn create_mqtt_client(
     let mqtt_config = mqtt_channel::Config::default()
         .with_session_name("c8y-log-plugin")
         .with_port(mqtt_port)
-        .with_subscriptions(topics);
+        .with_subscriptions(topics)
+        .with_last_will_message(get_health_status_down_message("c8y-log-plugin"));
```

Comment:
Minor: Same comment as above: move to a const.

Reply:
fixed